### DISCUSSION TITLE
I get a blank screen when I try to run the phaser example

### DISCUSSION THREAD

**Discussion by peteblank** [geckosio/geckos.io#133] [Help / Questions] [answered]
I managed to get it to run somehow but once I did I got a blank screen.
I'm using ubuntu ,chrome and node v14.x
![screenshot2222](https://user-images.githubusercontent.com/46042807/134212068-5d13aa84-e3a1-491b-bf95-a4eef9eb9eb1.png)

**Comment by yandeu** [comment, marked as answer]
I have just converted the example to ESM. Please try again.

**Comment by peteblank** [comment]
Thank you, I got it to run by installing babel though.

**Comment by peteblank** [comment]
This is very nice, I wish there was a tutorial though.
